src/lib/current_user.coffee

```diff
@@ -239,14 +239,10 @@ module.exports = class CurrentUser extends Backbone.Model
     @set followArtists: new Artists()
     @get('followArtists').fetchUntilEndInParallel(_.extend { url: url, data: data }, options)
 
+  # TODO: not sure what 'is_slumming' indicates
```

Comment:
😆 

Looks like it's from https://github.com/artsy/force-private/commit/fd1582a1b5c1151660b6030523e9c197b8a7ccfa but not much context there.